fs-repo-11-to-12/migration/swapper.go

```diff
@@ -19,6 +21,32 @@ var SyncSize uint64 = 20 * 1024 * 1024 // 20MiB
 // migration.
 var NWorkers int = 4
 
+func init() {
+	workerEnvVar := "IPFS_FS_MIGRATION_11_TO_12_NWORKERS"
+	syncSizeEnvVar := "IPFS_FS_MIGRATION_11_TO_12_SYNC_SIZE"
+	if nworkersStr, nworkerInEnv := os.LookupEnv(workerEnvVar); nworkerInEnv {
+		nworkers, err := strconv.Atoi(nworkersStr)
+		if err != nil {
+			panic(err)
+		}
+		if nworkers < 1 {
+			panic("number of workers must be at least 1")
+		}
```

Comment:
We could log an error and fallback to the defaults instead of panicking here, but this seems reasonable to let the user know they did something wrong.